vllm/worker/worker_base.py

```diff
@@ -115,9 +117,22 @@ def update_environment_variables(self, envs: Dict[str, str]) -> None:
 
     def init_worker(self, *args, **kwargs):
         """
-        Actual initialization of the worker class.
+        Actual initialization of the worker class, and set up
+         logging if required.
         Arguments are passed to the worker class constructor.
         """
+        if int(os.getenv("VLLM_TRACE_FUNCTION", "0")):
+            from vllm.logger import enable_trace_function_call
+            from vllm.utils import get_vllm_instance_id
+            tmp_dir = os.environ.get("TMPDIR") or "/tmp"
+            log_path = os.path.join(
+                tmp_dir, "vllm", get_vllm_instance_id(),
+                (f"VLLM_TRACE_FUNCTION_for_process_{os.getpid()}"
+                 f"_thread_{threading.get_ident()}_"
+                 f"at_{datetime.datetime.now()}.log").replace(" ", "_"))
```

Comment:
move filename to a variable

Reply:
fixed in [e9600f0](https://github.com/vllm-project/vllm/pull/4079/commits/e9600f09a0556e042a233bd52be79d145d2d0234)